libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/hooks/useDependentDateFiltersConfiguration.ts

```diff
@@ -64,21 +85,19 @@ export function useDependentDateFiltersConfiguration(
 
     const onConfigurationClose = useCallback(() => {
         setDependentDateFilters(originalState);
-    }, [originalState]);
+        setDependentCommonDateFilter(originalCommonDateFilterState);
+    }, [originalState, originalCommonDateFilterState]);
 
     const onDependentDateFiltersChange = useCallback(() => {
         // dispatch the command only if the configuration changed
         if (onDependentDateFiltersConfigurationChanged) {
-            const dateFilters: IDashboardAttributeFilterByDate[] = [];
-            dependentDateFilters.forEach((dependentDateFilter) => {
-                if (!dependentDateFilter.isSelected) {
-                    return;
-                }
-
-                dateFilters.push({
-                    filterLocalIdentifier: dependentDateFilter.localIdentifier,
-                });
-            });
+            const dateFilters = dependentDateFilters
+                .filter((filter) => filter.isSelected)
+                .map((filter) => ({
+                    filterLocalIdentifier: filter.localIdentifier,
+                    isCommonDate: filter.isCommonDate,
```

Comment:
Not sure where this bubble through but this could be ideal place to preface it with `!!` and have the `isCommonDate` non optional in the returned type.

Reply:
Updated it with `!!filter.isCommonDate`